tests/cross/src/test/scala/tests/highlight/Scala3DocumentHighlightSuite.scala

```diff
@@ -247,4 +247,24 @@ class Scala3DocumentHighlightSuite extends BaseDocumentHighlightSuite {
        |end extension""".stripMargin,
   )
 
+  check(
+    "enum-cases".tag(IgnoreScala2),
```

Comment:
You might want to ignore it for the nightlies for now, we should merge the change upstream to make it work there too. I can approve there as well

Reply:
should be good to go now